admin/admin-service/src/main/java/io/mosip/admin/packetstatusupdater/service/impl/PacketStatusUpdateServiceImpl.java

```diff
@@ -102,9 +103,9 @@ private PacketStatusUpdateResponseDto getPacketStatus(String rId) {
 
 			HttpHeaders packetHeaders = new HttpHeaders();
 			packetHeaders.setContentType(MediaType.APPLICATION_JSON);
-			StringBuilder urlBuilder = new StringBuilder();
-			urlBuilder.append(packetUpdateStatusUrl).append(SLASH).append(rId);
-			ResponseEntity<String> response = restTemplate.getForEntity(urlBuilder.toString(), String.class);
+			UriComponentsBuilder urlBuilder = UriComponentsBuilder.fromUriString(packetUpdateStatusUrl + SLASH)

```

Comment:
can you pls remove the SLASH concatenation.

Reply:
Done with the changes